src/lib/draw-and-pick.js

```diff
@@ -32,11 +33,10 @@ export function drawLayers({layers, pass}) {
 /* eslint-disable max-depth, max-statements */
 export function pickLayers(gl, {
   layers,
-  pickingFBO,
+  context,
```

Comment:
The way I see it, `context` is internal to the `LayerManager`/`Layer` classes. Probably cleaner to make this function take explicit args and not know anything about the context object.